demos/coreMQTT_Agent/CMakeLists.txt

```
@@ -0,0 +1,42 @@
+# FreeRTOS coreMQTT demos
+afr_demo_module(core_mqtt_agent)
+
+# Set the default demo that is enabled in the downloaded package.
+afr_set_demo_metadata(ID "CORE_MQTT_CONNECTION_SHARING_DEMO")
+afr_set_demo_metadata(DESCRIPTION "Examples that demonstrate the MQTT library")
+afr_set_demo_metadata(DISPLAY_NAME "coreMQTT Demos")
```

The metadata information for the coreMQTT Agent demos should be separate from the coreMQTT demos as they represented by 2 different demo targets now
```suggestion
afr_set_demo_metadata(DESCRIPTION "Examples that demonstrate the coreMQTT Agent library")
afr_set_demo_metadata(DISPLAY_NAME "coreMQTT Agent Demo")
```

---

I removed this for now. I'll add it back once we port the actual demo